README.md

```diff
@@ -62,10 +62,10 @@ Check out [plugins documentation](./docs/plugins.md).
 First, clone the repo via git:
 
 ```bash
-git clone https://github.com/KELiON/cerebro.git cerebro
+$ git clone https://github.com/KELiON/cerebro.git cerebro
```

Comment:
I understand that, but it is usually accepted in the console command to start a command prompt (for Unix-based system is a dollar), it is reasonable, although I will agree, complicates copying.